react-backend/client/src/components/admin/adminUsersPage.js

```diff
@@ -85,23 +83,43 @@ export default class AdminUsersPage extends Component {
   getActivitiesForChart = (activities) => {
     if (!activities) return [];
 
-    return activities.map(({ name, count }, index) => ({
-      value: count,
-      color: this.getColorPair(index).color,
-      highlight: this.getColorPair(index).highlight,
-      label: name,
-    }));
+    const activitiesData = {
```

Comment:
I'd recommend trying some other array methods here as you can minimise the amount of code you write. You can do something like this:

```js
    const primaryColors = ['#F7464A', '#46BFBD', '#FDB45C', '#949FB1', '#4D5360'];
    const secondaryColors = ['#FF5A5E', '#5AD3D1', '#FFC870', '#A8B3C5', '#616774'];

    const activitiesData = {
      labels: activities.map(el => el.name),
      datasets: [
        {
          data: activities.map(el => el.count),
          backgroundColor: primaryColors,
          hoverBackgroundColor: secondaryColors,
        },
      ],
    };
```

I've also added in an array of colours needed for `chartjs2` and removed the label in datasets as it's not needed 😊

Reply:
Colors should go in and/or come from the style guide.